.github/CODEOWNERS

```diff
@@ -2,20 +2,33 @@
 # Each line is a file pattern followed by one or more owners.
 # For details on acceptable file patterns, please refer to the [Github Documentation](https://help.github.com/articles/about-codeowners/)
 
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @XXXX will be requested for
+# review when someone opens a pull request.
+* @timmoreton
```

Comment:
Add a default owner such that a PR always gets an assignee. Appologies to put you here :).